src/streams.rs

```diff
@@ -0,0 +1,659 @@
+//! Defines types to use with the streams commands.
+
+use crate::{from_redis_value, FromRedisValue, RedisResult, RedisWrite, ToRedisArgs, Value};
+
+use std::collections::HashMap;
+use std::io::{Error, ErrorKind};
+
+// Stream Maxlen Enum
+
+/// Utility enum for passing `MAXLEN [= or ~] [COUNT]`
+/// arguments into `StreamCommands`.
+/// The enum value represents the count.
+#[derive(PartialEq, Eq, Clone, Debug, Copy)]
+pub enum StreamMaxlen {
+    /// Match an exact count
+    Equals(usize),
+    /// Match an approximate count
+    Aprrox(usize),
+}
+
+impl ToRedisArgs for StreamMaxlen {
+    fn write_redis_args<W>(&self, out: &mut W)
+    where
+        W: ?Sized + RedisWrite,
+    {
+        let (ch, val) = match *self {
+            StreamMaxlen::Equals(v) => ("=", v),
+            StreamMaxlen::Aprrox(v) => ("~", v),
+        };
+        out.write_arg("MAXLEN".as_bytes());
+        out.write_arg(ch.as_bytes());
+        val.write_redis_args(out);
+    }
+}
+
+/// Builder options for [`xclaim_options`] command.
+///
+/// [`xclaim_options`]: ./trait.StreamCommands.html#method.xclaim_options
```

Comment:
Change the file prefix to `trait.Commands.html`?

Reply:
Thanks, will change this